src/seedu/addressbook/data/person/Address.java

```diff
@@ -57,3 +75,51 @@ public boolean isPrivate() {
         return isPrivate;
     }
 }
+
+class Block {
```

Comment:
Missing header comment. All non-trivial classes should have javadoc format header comments.